redis/endpoint.bal

```diff
@@ -1058,7 +1064,7 @@ public client class Client {
     # + return - Boolean `true` if field is a new field in the hash and value was set. boolean false if
```

Comment:
Suggestion:
Start with a capital after the. full stop.

Noticed multiple places where this suggestion is applicable. Please have a look and apply if we do go ahead with the suggestion here.